put-a-jetpack-on-it/slides.md

```diff
@@ -0,0 +1,305 @@
+# Putting a Jetpack on your legacy codebase
+
+## Kurt Nelson
+### Android Platform at Pinterest
+
+Hey everyone, Kurt from the Pinterest Android Platform (or Android Experience) team.
+
+---
+What do I work on?
+
+# Platform Teams
+	- Mobile-wide issues
+	- Build, framework, testing
+	- Android, iOS, Web
+
+At many of the bigger tech companies with longstanding big apps, we spin up platform and infrastructure teams within engineering to concentrate on mobile-wide issues. Sister teams of ours for example are the mobile builds team and the test tooling team. There are also equivalents for iOS, Web, etc.
+
+---
+# Forward-looking big picture
+
+### Ensuring the stability, security and performance going forward
+
+### Library choices and upgrades
+
+One of our roles is to look at both the big picture and the future at the same time, hopefully heading external changes in best practices off enough that feature teams can write features and not have to worry about things like long term stability, performance, and library upgrades.
+
+---
+# We experiment early with new technologies
+
+This means we are also in charge of trying out new libraries, best practices, and patterns to see if they belong in our app.
+
+---
+# We pay attention to what is going on in the Android world
+	hi room.
+
+Platform teams are often in the room early, even possibly getting first access to libraries to play with.
+
+---
+# Handling Android Changes
+
+Of course, when Google makes large changes, we don’t as much have a choice of if we will use it but of when we will pick up the changes. 
```

Comment:
the sentence is a bit weird to read (it may just be me since i'm not a native speaker tough). I think for speaking is fine, but maybe for the blog can be cleaned a bit